src/utils.rs

```diff
@@ -100,6 +100,20 @@ pub fn get_socket_path() -> Result<PathBuf> {
                     legacy_path.to_string_lossy()
                 );
             }
+            #[cfg(windows)]
+            {
+                let kpnatmsg_str = "keepassxc\\".to_owned() + &std::env::var("USERNAME")? + "\\kpxc_server";
```

Comment:
`&std::env::var("USERNAME")?`

If we `?` here, when USERNAME doesn't exist, I think it'll return Err immediately and [1] won't get a chance to run?

[1] https://github.com/pitdicker/git-credential-keepassxc/blob/f3f312200fc7a5d918ed51922dcddffda4a6c09d/src/utils.rs#L129-L131